uber/site_sections/emails.py

```diff
@@ -71,3 +71,25 @@ def test_email(self, session, subject=None, body=None, from_address=None, to_add
     def approve(self, session, subject):
         session.add(ApprovedEmail(subject=subject))
         raise HTTPRedirect('pending?message={}', 'Email approved and will be sent out shortly')
+
+    def emails_by_interest(self):
+        return {}
+
+    @csv_file
+    def emails_by_interest_csv(self, out, session, *args, **kargs):
+        """
+        Generate a list of emails of attendees who match one of c.INTEREST_OPTS
+        (interests are like "LAN", "music", "gameroom", etc.
+
+        This is intended for use to export emails to a third-party email system, like MadMimi or Mailchimp
+        """
+        assert 'interests' in kargs
```

Comment:
The way you have this now, they'll get a 500 error if they submit the form without any checkboxes checked.  A better approach would probably be

```python
if 'interests` not in params:
    raise HTTPRedirect('email_by_interest?message={}', 'You must select at least one interest')
```

You'll also need to add a ``message=''`` parameter to that page.